disassemblers/adreno/__init__.py

```diff
@@ -8,7 +8,7 @@ def disasm(buf):
   global fxn
   if fxn is None:
     shared = pathlib.Path(__file__).parent / "disasm.so"
-    if not os.path.isfile(shared):
+    if not shared.is_file():
```

Comment:
Pretty sure. Change only affects the if condition. `Path.is_file()` and `os.path.isfile() have the same implementation -> They use `stat.st_mode` to check whether it is a regular file.